upgrade-1.x-to-2.x.sh

```diff
@@ -76,6 +88,14 @@ else
     esac
 fi
 
+# Find boot path
+boot_path=$(findmnt -n --raw --evaluate --output=target /dev/mmcblk0p1)
+if [ -z "${boot_path}" ]; then
+    log ERROR "Cannot find where is /dev/mmcblk0p1 mounted"
```

Comment:
This is a hard assumption because there will be devices with other block device names. Can we generalise this? In the past we were mounting the boot partition in /boot while now we do it in /mnt/boot. So these are the only mount locations possible for boot partition.